task_processing/plugins/kubernetes/task_config.py

```diff
@@ -304,6 +304,17 @@ def __invariant__(self) -> Tuple[Tuple[bool, str], ...]:
         initial=10.0,
         factory=float,
         invariant=lambda d: (d > 0, 'disk > 0'))
+
+    request_cpus = field(
+        type=float,
+        factory=float)
+    request_memory = field(
+        type=float,
+        factory=float)
+    request_disk = field(
+        type=float,
+        factory=float)
```

Comment:
imo, `$RESOURCE_request` would read a tad better :)

that said, for my suggestion above about giving these a null initial value:
```suggestion
    request_cpus = field(
        type=(float, type(None)),
        initial=None,
    )
    request_memory = field(
        type=(float, type(None)),
        initial=None,
    )
    request_disk = field(
        type=(float, type(None)),
        initial=None,
    )
```

Reply:
Flipping the name totally makes sense, i mostly worded it as request_x because of how it is in the structure itself.